src/components/HiddenTaskList/HiddenTaskList.jsx

```diff
@@ -0,0 +1,89 @@
+const React = require('react')
+const { connect } = require('react-redux')
+
+const HiddenTaskListItem = require('../HiddenTaskListItem')
+const hookUpStickyNav = require('../hookUpStickyNav')
+const TaskVisibility = require('../../models/TaskVisibility')
+
+class HiddenTaskList extends React.Component {
+  onRestoreClick(event) {
+    event.currentTarget.blur() // defocus button
+    this.props.dispatch({ type: 'TASKS_RESTORE' })
+  }
+
+  cancel(event) {
+    event.preventDefault()
+    this.props.cancel()
+  }
+
+  navigation(tasks) {
+    if (tasks.length < 1) {
+      return
+    }
+    return (
+      <nav className="controls-container has-text-right">
+        <label className="label">With selected:</label>
+        <button
+          type="button"
+          onClick={e => this.onRestoreClick(e)}
+          className="control button is-link"
+          id="restore-button"
+          title="Restore selected"
+        >↩️</button>
+      </nav>
+    )
+  }
+
+  emptyListMessage(tasks) {
+    if (tasks.length > 0) {
+      return
+    }
+    return (
+      <p className="content">
+        There are no issues or pull requests that have been snoozed, archived,
+        or ignored for this filter.
+      </p>
+    )
+  }
+
+  render() {
+    const { activeFilter } = this.props
```

Comment:
Yes! This line is equivalent to:

```javascript
const activeFilter = this.props.activeFilter
```

This is part of Glorious Modern JavaScript ™️ so you can use it to suck out properties from objects wherever you like. This is also what's happening in GitHub.js on this line:

```javascript
return this.get(urlPath).then(({ items }) => items.map(d => getTask(d)))
```